benchmarks/run_blocking_ab.ps1

```
@@ -358,6 +369,32 @@ foreach ($a in $ArmList) { Write-Host "  $(Arm-Csv $a)" }
 # Every arm against the baseline. `default` is the baseline when present -- it is
 # what MTL5 ships, so every other arm is a proposed change to it.
 $Base = if ($ArmList -contains "default") { "default" } else { $FirstArm }
+# Arms that derived the SAME kc/mc measure the same thing, so comparing them
+# checks the SESSION rather than the code -- and the analyzer fails it when they
+# disagree systematically (#430).
+function Blocking-Of($arm) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Blocking-Of' uses an unapproved verb.

(PSUseApprovedVerbs)